src/bundle/Resources/views/themes/admin/system_info/my_ibexa.html.twig

```diff
@@ -0,0 +1,70 @@
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.version'|trans|desc('Version') }}</td>
+                    <td class="ez-table__cell">{{ info.release }}</td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.is_dxp'|trans|desc('Is DXP') }}</td>
+                    <td class="ez-table__cell">
+                        <svg class="ez-icon ez-icon--small">
+                            <use xlink:href="{{ ez_icon_path((info.isEnterprise == true) ? 'checkmark' : 'discard') }}"></use>
+                        </svg>
+                    </td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.is_commerce'|trans|desc('Has Commerce') }}</td>
+                    <td class="ez-table__cell">
+                        <svg class="ez-icon ez-icon--small">
+                            <use xlink:href="{{ ez_icon_path((info.isCommerce == true) ? 'checkmark' : 'discard') }}"></use>
+                        </svg>
+                    </td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.is_trial'|trans|desc('Is Trial') }}</td>
+                    <td class="ez-table__cell">
+                        <svg class="ez-icon ez-icon--small">
+                            <use xlink:href="{{ ez_icon_path((info.isTrial == true) ? 'checkmark' : 'discard') }}"></use>
+                        </svg>
+                    </td>
+                </tr>
+                <tr class="ez-table__row">
+                    <td class="ez-table__cell">{{ 'ibexa.stability'|trans|desc('Stability') }}</td>
+                    <td class="ez-table__cell">{{ info.stability }}</td>
+                </tr>
+                {% if info.isEnterprise == true %}
+                    <tr class="ez-table__row">
+                        <td class="ez-table__cell">{{ 'ibexa.eom'|trans|desc('End of Maintenance') }}*</td>
+                        <td class="ez-table__cell">
+                            {{ info.endOfMaintenanceDate is empty ? '' : info.endOfMaintenanceDate|date('F Y') }}
+                        </td>
+                    </tr>
+                    <tr class="ez-table__row">
+                        <td class="ez-table__cell">{{ 'ibexa.eol'|trans|desc('End of Life') }}*</td>
+                        <td class="ez-table__cell">
+                            {{ info.endOfLifeDate is empty ? '' : info.endOfLifeDate|date('F Y') }}
+                        </td>
+                    </tr>
+                    <tr class="ez-table__row">
+                        <td colspan="2">
+                            <hr/>
+                            * {{ 'ibexa.read_more'|trans|desc('You can read more about Service Life for Ibexa DXP with a business license') }}
```

Comment:
You could put whole link inside translation, there are probably some other languages where link wo'nt be last word

Reply:
Not sure if I get what you meant, so I explicitly mentioned the URL in the link. Having the whole text as a link doesn't look good. If you suggest rephrasing the whole sentence I am open to that.

Reply:
Maybe reference to use `%link%` inside  the string?

Reply:
As agreed on Slack, we can leave it as-is for now, possible iterations might be done as followups.